vignettes/Example1-Ostats-univariate.Rmd

```diff
@@ -220,13 +222,121 @@ Ostats_plot(plots = siteID,
             traits = trait, 
             overlap_dat = small_mammal_Ostats, 
             use_plots = sites2use, 
-            name_x = expression(paste('log'[10], ' body mass (g)')), 
+            name_x = 'log10 Body Mass (g)', 
             means = TRUE)
 ```
 
 The graphs to the left show the intraspecific trait variation as illustrated by density curves, and the graphs to the right show mean trait values. For all graphs probability density is shown on the y-axis and log-transformed body mass on the x-axis. Each color represents a different species and colors are consistent across graphs. The top row illustrates trait values for HARV (Harvard Forest), whereas the bottom row illustrates trait values for JORN (Jornada). The left column of graphs also show the raw O-statistic for the entire community. Note that there is more overlap in body mass for HARV than for JORN. 
 
-Note that currently there is no functionality within the `Ostats` package to graph the outputs of `Ostats()` when it is applied to circular data.
+###Example: Plotting Univariate Circular O-Statistics
+
+If trait values are circular, a circular kernel density estimate for
+each species is plotted on a polar coordinate plot. If trait values are
+both circular and discrete, a "sunburst" plot is returned.
+
+The following example plots the ant data for chambers 1 and 2. Note that when plotting circular data for the ant example, the arguments 'circular' and 'discrete' should be set to TRUE because ant behavioral traits are measured with periodic units of discrete time (i.e., hours in a day). 
+
+```{r out.width="80%", warning=FALSE, error=FALSE, fig.align="center", fig.height=4.5, fig.width=7}
+siteID <- factor(ant_data$chamber)
+taxonID <- factor(ant_data$species)
+trait <-  as.matrix(ant_data[, 'time', drop = FALSE])
+
+
+Ostats_plot(plots = siteID, 
+            sp = taxonID, 
+            traits = trait, 
+            overlap_dat = circular_example, 
+            name_x = 'Time (hours)', 
+            circular = TRUE,
+            discrete = TRUE)
+```
+
+
+## Boostrapping
+
+### Example for generating bootstrap of linear univariate O-statistics
+
+This is an illustration of how to bootstrap O-statistics using the example data on small mammal body mass from Harvard Forest (HARV) and Jornada (JORN). It is written without any tidyverse packages, other than ggplot2, to not add any new dependencies.
```

Comment:
You can get rid of the sentence about not using tidyverse, I just said that for our own internal communication but I do not think it needs to be in the vignette

Reply:
deleted this last sentence referring to tidyverse